lms/djangoapps/cohort_management_dashboard/urls.py

```diff
@@ -0,0 +1,19 @@
+"""
+Defines the URL routes for this app.
+"""
+from django.conf.urls import url
+
+from openedx.core.djangoapps.course_groups.views import users_in_cohort
+from . import views
+
+app_name = u'lms.djangoapps.cohort_management_dashboard'
+urlpatterns = [
+    url(r'^dashboard/(?P<cohort_id>\d+)$', views.DashboardIndex.as_view(), name='dashboard_with_cohort'),
+    url(r'^dashboard', views.DashboardIndex.as_view(), {'cohort_id': None}, name='dashboard'),
+    # Note(yura.braiko@raccongang.com) for some reason rout for all_user_list in edx-platform/lms/urls.py doesn't work.
```

Comment:
Are we going to know the reason?

Reply:
Yes. Cohort's URL contains 2 patterns, which match to the same expression and as a result users_in_cohort become unreachable.